multimap/src/test/java/org/infinispan/multimap/impl/DistributedMultimapCacheTest.java

```diff
@@ -509,9 +506,7 @@ protected void assertKeyValueFoundInOwners(String key, Person value) {
    @Override
    protected void assertOwnershipAndNonOwnership(Object key, boolean allowL1) {
       for (Cache cache : caches) {
-         Wrapper keyWrapper = cache.getAdvancedCache().getKeyWrapper();
-         Encoder keyEncoder = cache.getAdvancedCache().getKeyEncoder();
-         Object keyToBeChecked = keyEncoder != null && keyWrapper != null ? EncodingUtils.toStorage(key, keyEncoder, keyWrapper) : key;
+         Object keyToBeChecked = cache.getAdvancedCache().getKeyDataConversion().getEncoder().toStorage(key);
```

Comment:
I believe this line should be `Object keyToBeChecked = cache.getAdvancedCache().getKeyDataConversion().getEncoder().toStorage(key);` instead.